test/throttle-binding-behavior.spec.js

```diff
@@ -143,7 +144,7 @@ describe('ThrottleBindingBehavior', () => {
       bindingEngine.observerLocator.eventManager,
       'mousemove',
       bindingEngine.parseExpression(`handleMouseMove($event) & throttle:${delay}`),
-      false,
+      delegationStrategy.none,
       false,
```

Comment:
Great, I read about rebase but not sure if it was the right thing to do, git documentation was quite a puzzle for me plus I'm still newbie. Tks.

Updates:
- Rebased
- Added test for all `delegationStrategy`
- Fixed unbind